doc/api/http.md

```diff
@@ -974,7 +974,11 @@ const headerNames = response.getHeaderNames();
 added: v7.7.0
 -->
 
-* Returns: {Object}
+* Returns: {Object}  
+  *Note*: The object returned by the `response.getHeaders()` method _does not_
+  prototypically extend the JavaScript `Object`. This means that typical`Object`
+  methods such as `obj.toString()`, `obj.hasOwnProperty()`, and others are not
+  defined and *will not work*. As well `obj instanceof Object === false`.
```

Comment:
Either make the last bit a complete sentence, or fold it into the previous sentence.